olarewajuakeemope/document-manager
# Feature/backend/signup Logic

**What does this PR do?**

- completes the necessary backend logic for the signup implementation

**Description of Task to be completed?**

- adds new user to the database

- creates a jwt token for the new user

- returns the user data and token to the client side on successful signup

**How should this be manually tested?**

- after installing the application locally, run the following command:
 `npm run start:server`

- visit `http://localhost:8080` on your browser, you should be guided to the signup page

- on successful signup, the jwt token is stored in the local storage as jwtToken property

- page currently doesn't redirect after successful signup but local storage status can be verified from browser console
